src/Login/Login.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import Header from '../Header/Header.js'
+import './Login.css'
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      error: ''
+    };
+  }
+
+  updateText = (event) => {
+    this.setState({[event.target.id]: event.target.value})
+  }
+
+  generateCredentials = () => {
+    const emailInput = this.state.email;
+    const passwordInput = this.state.password;
+
+    return {
+      email: emailInput,
+      password: passwordInput
+    }
+  }
+
+  login = () => {
+    const credentials = this.generateCredentials();
+
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(credentials)
+    }).then(response => response.json())
+      .then(data => {
+        this.props.updateCurrentUser(data.user)
+        if (data.user) {
+          this.props.displayHomePage();
+        } else {
+          this.setState({ error: 'Uh oh' });
+          console.log(this.state.error);
+        }
```

Comment:
I want to prioritize this for the refactor - I'm sure this logic can be cleaned up